alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/resthandler/RestIndexMonitorAction.kt

```diff
@@ -78,10 +80,23 @@ class RestIndexMonitorAction(
 ) : BaseRestHandler(settings) {
 
     private var scheduledJobIndices: ScheduledJobIndices
+
     @Volatile private var maxMonitors = ALERTING_MAX_MONITORS.get(settings)
     @Volatile private var requestTimeout = REQUEST_TIMEOUT.get(settings)
     @Volatile private var indexTimeout = INDEX_TIMEOUT.get(settings)
 
+    companion object {
+        var scheduledJobSchemaVersion: Int? = null
```

Comment:
with this other classes can set values to this variable. You want to add `private set` to prevent it from publicly modifiable. Kotlin syntax should be 

```kotlin
var scheduledJobSchemaVersion: Int? = null
    private set
```

https://kotlinlang.org/docs/reference/properties.html

Reply:
Good suggestion. Will add `private set`